### PR TITLE
Request PIDs for kinX USB Hub/Keyboard Controller

### PR DESCRIPTION
**Pull request by stapelberg**

### CODE REVIEW

**Comment by stapelberg** [comment]
The travis failure seems unrelated to me.

File: 1209/345C/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Keyboard Controller
+owner: kinX
+license: CERN Open Hardware Licence v1.2
+site: https://michael.stapelberg.ch/posts/2018-04-17-kinx/
+source: https://github.com/kinx-project/mk66f-hw
+---
```
Comment (Arachnid) [inline 1209/345C/index.md]: Can you please link to the firmware source in the body of the file too?

Comment (stapelberg) [reply 1209/345C/index.md]: Sorry, not sure I follow. Your comment is literally one line above a link to the firmware in the body of the file, so maybe I misunderstand what you mean? If you meant to comment on the other file, there is no firmware required for the USB hub IC which that project uses, but its parameters are customizable, so a custom VID/PID still makes sense.

Thanks,

Comment (Arachnid) [reply 1209/345C/index.md]: Sorry, I managed to miss that somehow.

**Comment by Arachnid** [comment]
Can you please rebase your PR against the current head?

**Comment by stapelberg** [comment]
Done.

**Pull request merged**